test/copy.jl

```diff
@@ -63,6 +63,10 @@ end
         @test B[5:7,2:3] == A
         B[5:7,2:3] .= 0
         @test all(x->x==0, B)
+        copyto!(B, CartesianIndices((3:2:7,1:2:3)), A, RA)
+        @test B[3:2:7,1:2:3] == A
+        copyto!(B, CartesianIndices((7:-1:5,3:-1:2)), A, CartesianIndices(map(reverse, axes(A))))
```

Comment:
Maybe add one more test, where the indices of `B` have a positive step and those of `A` have a negative step? That should complement the current tests where the step is positive or negative for both.

Reply:
Added.